modules/ampir/main.nf

```diff
@@ -0,0 +1,43 @@
+process AMPIR {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "conda-forge::r-ampir=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-ampir:1.1.0':
+        'quay.io/biocontainers/r-ampir:1.1.0' }"
+
+    input:
+    tuple val(meta), path(faa)
+    val cut_off
+    val model
+    val output_name
+
+    output:
+    tuple val(meta), path(output_name)  , emit: amps
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    #!/usr/bin/env Rscript
+    library(ampir)
+
+    protein_seqs <- read_faa("${faa}")
+    prediction <- predict_amps(protein_seqs, model = '$model')
```

Comment:
Looking at the docs, wouldn't we also want the probability statistics in the table produced by `predict_amps`?

This actually has the values with probabilities so would be more useful for evaluation too (prior filtering)

https://github.com/Legana/ampir

```suggestion
    prediction <- predict_amps(protein_seqs, model = '$model')
    write.csv(prediction, file = "${prefix}.csv")
```